06.bowling_object/game.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+
+class Game
+  NUM_OF_FRAMES = 10
+
+  def initialize(score)
+    @score = score
+  end
+
+  def calculate_total_score
+    marks = @score.split(',')
+    frames = generate_marks_by_frame(marks).map do |marks_in_frame|
+      Frame.new(marks_in_frame)
+    end
+
+    (0..NUM_OF_FRAMES - 1).sum do |frame_idx|
+      calculate_frame_score(frame_idx + 1, *frames[frame_idx, 3])
+    end
```

Comment:
```suggestion
    frames.each_with_index.sum do |frame, i|
      next_frame = ...
      after_next_frame = ...
      frame.calculate_score(next_frame, after_next_frame)
    end
```

みたいにしてFrameクラスにスコア計算させた方がオブジェクト指向らしくなりそうです。